From 1c2215539b6fc902007acb74838c26980a8582a3 Mon Sep 17 00:00:00 2001
From: Anders Kaseorg <andersk@mit.edu>
Date: Wed, 10 Apr 2013 06:26:00 -0400
Subject: [PATCH] Get rid of all our embedded copies of Module::Install

Signed-off-by: Anders Kaseorg <andersk@mit.edu>
---
 perl/modules/Makefile.am | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/perl/modules/Makefile.am b/perl/modules/Makefile.am
index e6bf04b..3d31e69 100644
--- a/perl/modules/Makefile.am
+++ b/perl/modules/Makefile.am
@@ -1,4 +1,4 @@
 MODULES = Jabber IRC WordWrap Twitter Facebook
 
-EXTRA_DIST = $(MODULES:=/Makefile.PL) $(MODULES:=/inc) $(MODULES:=/lib)
+EXTRA_DIST = $(MODULES:=/Makefile.PL) $(MODULES:=/lib)
 
@@ -7,6 +7,6 @@ $(MODULES): %: %/Makefile
 	(cd $* && $(MAKE) $*.par)
 
 MODULES_MAKEFILE = $(MODULES:=/Makefile)
-$(MODULES_MAKEFILE): %/Makefile: %/Makefile.PL
+$(MODULES_MAKEFILE): %/Makefile: %/Makefile.PL ../lib/Module/Install/BarnOwl.pm
 	$(AM_V_GEN)(cd $* && perl -I../../lib Makefile.PL)
 
-- 
1.8.4.4

